pilot/tools/types.go.tmpl

```diff
@@ -0,0 +1,142 @@
+// Code generated by generate.sh. DO NOT EDIT!
```

Comment:
update this to point to the new tool please

Reply:
Really, this could/should be a [`//go:generate`](https://blog.golang.org/generate) directive.